tests/lib/snaps/basic-run/meta/snap.yaml

```diff
@@ -0,0 +1,6 @@
+name: basic-run
```

Comment:
There is a snap called `test-snapd-sh` that lets you run sh directly without any fuss. Would it suffice?

Reply:
Not sure, what I need is a env variable inside the `command: ` part of the app. I could add that to another test snap of course if you think this is overkill.